### PR TITLE
Enable EM_TEST_VERBOSE when running test_poppler under fastcomp

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by kripken** [comment]
It is a useful test, on the other hand.

Do you see it take 10 minutes locally? I'd like to at least check there isn't something obviously wrong on CI before disabling a test like that.

**Comment by sbc100** [comment]
Its take a long time locally too yes.  I don't know why we've not seen to fail more often.    I looked back and some historical success run and these indivudual tests have always taken over 1000 seconds (~20 mins) each.    Maybe something with circle ci changed?

If you prefer I can just add EM_TEST_VERBOSE so the output of the configure phase (the things  that takes forever) appears in the stdout?

**Comment by kripken** [comment]
Verbose output might be best, yes. I just don't want us to lose important coverage for fastcomp just before we remove it.

**Comment by sbc100** [comment]
Done

File: .circleci/config.yml [hunk 1]
```diff
@@ -425,11 +425,13 @@ jobs:
           test_targets: "asm*.test_j* asm*.test_k* asm*.test_l* asm*.test_m* asm*.test_n* asm*.test_o*"
   test-fastcomp-p:
     executor: bionic
+    environment:
+      # Without this test_poppler can cause timeouts because it takes more then
+      # 10 minutes without outputing anything.
+      EM_BUILD_VERBOSE: "2"
```
Comment (kripken) [inline .circleci/config.yml:431]: this affects all tests in this suite, but I guess there aren't many in it?

did you notice if adding this makes the build slower (while making it more robust to timeouts)?

**Comment by sbc100** [comment]
Comparing to a previous run:

https://app.circleci.com/pipelines/github/emscripten-core/emscripten/6408/workflows/7bf4af5e-494d-4afb-b6f5-16442783e07b/jobs/329231/steps

time: 19:07
lines: 274

This run:

time: 19:10
lines: 6869

So I does add a lot of output.. but its doesn't seem to effect the overall time which is most taken up running autoconf tests.

**Pull request merged**